source/Octopus.Cli/Commands/Package/PackCommand.cs

```diff
@@ -31,36 +31,36 @@ public class PackCommand : CommandBase, ICommand, ISupportFormattedOutput
         SemanticVersion version;
         IPackageBuilder packageBuilder;
         string allReleaseNotes;
-        string compressionLevel;
+        PackageCompressionLevel packageCompressionLevel;
 
         public PackCommand(IOctopusFileSystem fileSystem, ICommandOutputProvider commandOutputProvider) : base(commandOutputProvider)
         {
             this.fileSystem = fileSystem;
 
             var common = Options.For("Advanced options");
-            common.Add("include=", "[Optional, Multiple] Add a file pattern to include, relative to the base path e.g. /bin/*.dll - if none are specified, defaults to **", v => includes.Add(v));
-            common.Add("overwrite", "[Optional] Allow an existing package file of the same ID/version to be overwritten", v => overwrite = true);
+            common.Add<string>("include=", "[Optional, Multiple] Add a file pattern to include, relative to the base path e.g. /bin/*.dll - if none are specified, defaults to **", v => includes.Add(v));
+            common.Add<bool>("overwrite", "[Optional] Allow an existing package file of the same ID/version to be overwritten", v => overwrite = true);
 
             var zip = Options.For("Zip packages");
-            zip.Add("compressionLevel=", "[Optional] Set compression level of package: none, fast, optimal (default).", c => compressionLevel = c);
+            zip.Add<PackageCompressionLevel>("compressionLevel=", "[Optional] Set compression level of package: none, fast, optimal (default).", c => packageCompressionLevel = c);
```

Comment:
Should the list of valid values be generated from the enum now? E.g. https://github.com/OctopusDeploy/OctopusCLI/blob/7cbb503ac8720481735bf62b5b004719a5722178/source/Octopus.Cli/Commands/InstallAutoCompleteCommand.cs#L25